Project/app/src/main/java/org/ionproject/android/common/model/Exam.kt

```diff
@@ -30,15 +30,15 @@ class Exam(
  */
 fun createExam(properties: ComponentProperties): Exam {
     val uid = properties.uid.value[0]
-    val summary = properties.summary?.value?.get(0)
-    val description = properties.description?.value?.get(0)
-    val categories = properties.categories?.value?.get(0)
+    val summary = properties.summary.value[0]
```

Comment:
This approach is used all over the place (unconditional indexation of the list). If this is the case, what is the point? We should discuss this.